main.nf

```diff
@@ -318,7 +318,7 @@ process TO_GENE_SYM {
 }
 
 
-process PHRANK_SCORING {
+process GENESYM_TO_PHRANK {
```

Comment:
this process calculate phenotype rank of genes, so we need to change the another name.

Reply:
does not "phenotype rank" mean phrank ? then is not it a legit name

Reply:
I was pointing genesym might not be the one we could put here.

Reply:
Oh this name is suggested with the rule of `INPUT_TO_OUTPUT`, (which might look not smart)

Currently I am understanding this process takes the input of `gene` which was generated by `ENSEMBL_TO_GENESYM`, so I thought `genesym` is the primary input of this process, and that's how this name was suggested.